[2025-02-09T06:31:45.626-0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-09T06:31:45.645-0500] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_reporte_como_vamos.bot_ex_bbva scheduled__2025-02-08T11:30:00+00:00 [queued]>
[2025-02-09T06:31:45.656-0500] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_reporte_como_vamos.bot_ex_bbva scheduled__2025-02-08T11:30:00+00:00 [queued]>
[2025-02-09T06:31:45.657-0500] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2025-02-09T06:31:45.673-0500] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): bot_ex_bbva> on 2025-02-08 11:30:00+00:00
[2025-02-09T06:31:45.680-0500] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=71768) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-09T06:31:45.682-0500] {standard_task_runner.py:72} INFO - Started process 71769 to run task
[2025-02-09T06:31:45.683-0500] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_reporte_como_vamos', 'bot_ex_bbva', 'scheduled__2025-02-08T11:30:00+00:00', '--job-id', '87', '--raw', '--subdir', 'DAGS_FOLDER/etl_como_vamos_dag.py', '--cfg-path', '/tmp/tmp0qm1mnp9']
[2025-02-09T06:31:45.684-0500] {standard_task_runner.py:105} INFO - Job 87: Subtask bot_ex_bbva
[2025-02-09T06:31:45.738-0500] {task_command.py:467} INFO - Running <TaskInstance: etl_reporte_como_vamos.bot_ex_bbva scheduled__2025-02-08T11:30:00+00:00 [running]> on host 21f1c1b48730
[2025-02-09T06:31:45.833-0500] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_reporte_como_vamos' AIRFLOW_CTX_TASK_ID='bot_ex_bbva' AIRFLOW_CTX_EXECUTION_DATE='2025-02-08T11:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-08T11:30:00+00:00'
[2025-02-09T06:31:45.834-0500] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-09T06:31:45.850-0500] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-09T06:31:45.851-0500] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/scripts/bot_etl_ex_bbva.py']
[2025-02-09T06:31:45.862-0500] {subprocess.py:99} INFO - Output:
[2025-02-09T06:31:46.847-0500] {subprocess.py:106} INFO - 2025-02-09 06:31:46,847 - bot_etl_ex_bbva - INFO - Bot ETL Exito-BBVA started
[2025-02-09T06:31:47.320-0500] {subprocess.py:106} INFO - 2025-02-09 06:31:47,320 - bot_etl_ex_bbva - INFO - Conexi√≥n al Selenium Hub establecida correctamente
[2025-02-09T06:31:49.295-0500] {subprocess.py:106} INFO - 2025-02-09 06:31:49,295 - bot_etl_ex_bbva - INFO - Credentials input success
[2025-02-09T06:31:50.687-0500] {subprocess.py:106} INFO - 2025-02-09 06:31:50,687 - bot_etl_ex_bbva - INFO - Click in login button
[2025-02-09T06:32:00.709-0500] {subprocess.py:106} INFO - 2025-02-09 06:32:00,709 - bot_etl_ex_bbva - INFO - Loging success
[2025-02-09T06:32:01.769-0500] {subprocess.py:106} INFO - 2025-02-09 06:32:01,768 - bot_etl_ex_bbva - INFO - Input file succes
[2025-02-09T06:32:11.180-0500] {subprocess.py:106} INFO - 2025-02-09 06:32:11,179 - bot_etl_ex_bbva - INFO - Click in Cargar
[2025-02-09T06:32:16.803-0500] {subprocess.py:106} INFO - 2025-02-09 06:32:16,802 - bot_etl_ex_bbva - INFO - bot_etl_ex_bbva finish execution
[2025-02-09T06:32:16.940-0500] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-09T06:32:16.975-0500] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-09T06:32:16.976-0500] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_reporte_como_vamos, task_id=bot_ex_bbva, run_id=scheduled__2025-02-08T11:30:00+00:00, execution_date=20250208T113000, start_date=20250209T113145, end_date=20250209T113216
[2025-02-09T06:32:17.020-0500] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-09T06:32:17.044-0500] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-09T06:32:17.047-0500] {local_task_job_runner.py:245} INFO - ::endgroup::
